frontend/src/Dashboard/CrossingDetailPage/CrossingDetails.js

```diff
@@ -1,6 +1,34 @@
 import React, { Component } from 'react';
+import gql from 'graphql-tag';
+import { graphql } from 'react-apollo';
 
 class CrossingDetails extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      name: props.crossing.name,
+      description: props.crossing.description
+    };
+  }
+
+  updateCrossing(e) {
+    debugger;
+    this.props.updateCrossingMutation({
+      variables: {
+        crossingId: this.props.crossing.id,
+        name: this.state.name,
+        description: this.state.description
+      }
+    })
+    .then(({ data }) => {
+      alert(data);
```

Comment:
This returns `[Object object]` for me. Seems like a placeholder for a success msg, but could it be more helpful in the meantime?

Reply:
replaced with a todo